tests-semanticdb/src/test/scala/scala/meta/tests/semanticdb/MetacScalaLibrary.scala

```diff
@@ -5,33 +5,38 @@ import scala.meta.internal.io.FileIO
 import scala.meta.io.AbsolutePath
 import scala.meta.metac._
 import scala.meta.tests.BuildInfo
-import scala.meta.tests.SkipWindows
-import scala.meta.tests.Slow
 
 import java.io.File
 import java.net.URLClassLoader
 import java.nio.file.Files
 import java.nio.file.Paths
 
 import scala.concurrent.duration
+import scala.util.Properties
 
 import munit.FunSuite
 
 class MetacScalaLibrary extends FunSuite {
 
   override val munitTimeout = new duration.FiniteDuration(3, duration.MINUTES)
 
-  test("compile scala-library".tag(Slow).tag(SkipWindows)) {
+  test("compile scala-library") {
+    assume(!Properties.isWin, "Test is not checked on windows")
```

Comment:
and how do you ensure that?

Reply:
This tests is skipped if the assumption is false